### DISCUSSION TITLE
My academic affiliation could not be verified

### DISCUSSION THREAD

**Discussion by vivekboora** [community/community#50158] [GitHub Education]
I upload my college ID where Date of Validity of the ID card was clearly mentioned but still Github said in the email that they could not verify my academic affiliation based on the information I provided. Can anybody guide me ?

**Comment by Ntshangase** [comment]
Hi,

I would suggest you check out: https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved

**Reply by vivekboora** [reply]
Thank you @Ntshangase for the suggestion. I uploaded my admission letter instead of uploading the College ID but chose College ID in the options and it worked!!

**Reply by Ntshangase** [reply]
Those are good news. If you don't mind please " mark as answer " the response that helped you, to help other people find this solution too and It also helps me get that galaxy brain badge.

**Comment by samus-aran** [comment]
Hi @vivekboora 👋 

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

**Reply by vivekboora** [reply]
Thank you @samus-aran  for the suggestion. I uploaded my admission letter instead of uploading the College ID but chose College ID in the options and it worked!!
